### PR TITLE
added curated list of community migration approaches

### PR DESCRIPTION
**Pull request by lwluc**
# Camunda Community Hub Pull Request Template

Started building a curated list of interesting approaches which should help to during migration.

## Description

List of interesting approaches which should help to migrate. It might be an architectural approach or just tips and tricks / experiences to keep in mind.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an existing open issue)
- [ ] New feature (non-breaking change which adds functionality to an extension)
- [ ] Breaking change (fix or feature that would cause existing functionality of an extension to change)
- [x] Documentation update (changes made to an existing piece of documentation)

## Checklist:

- [x] My code adheres to the syntax used by this extension.
- [ ] My pull request requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **Camunda Community Hub** documentation.
- [x] I have read the **Pull Request Process** documentation.
- [ ] I have added or suggested tests to cover my changes suggested in this pull request.
- [ ] All new and existing CI/CD tests passed.
- [ ] I will /assign myself this issue, and add the relevant [issue labels] to it if they are not automatically generated by Probot.
- [ ] I will tag @camunda-community-hub/devrel in a new comment on this issue if 30 days have passed since my pull request was opened and I have not received a response from the extension's maintainer.

### CODE REVIEW

**Comment by ingorichtsmeier** [comment]
Thank you @lwluc for your contribution.

It is worth to collect different approaches here.

I will add the example from phactum as well: https://github.com/vanillabp/simple-vanillabp-demo.

**Comment by ingorichtsmeier** [comment]
@lwluc, as you link to a pull request in your repo, do you take care when you merge it to adjust the link here?

File: migration-approaches/Curated-List.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Curated list of community migration approaches
+
+List of interesting approaches which should help to migrate. It might be an [architectural approach](#architectural-approaches) or just tips and tricks / experiences to keep in mind.
+
+## Architectural approaches
+
+* [Leveraging Clean Architecture for Migration](https://github.com/lwluc/camunda-ddd-and-clean-architecture/pull/1)
```
Comment (ingorichtsmeier) [inline migration-approaches/Curated-List.md:7]: Could you add a sentence or two about the context and your special approach to migration?

**Review by ingorichtsmeier — CHANGES_REQUESTED** [review]

File: migration-approaches/Curated-List.md (hunk 1 above)
Comment (lwluc) [reply migration-approaches/Curated-List.md]: Hey @ingorichtsmeier I added a short description. I hope it introduces the approach well. What do you think?

Comment (ingorichtsmeier) [reply migration-approaches/Curated-List.md]: Looks good!

**Review by ingorichtsmeier — APPROVED** [review]
LGTM

**Pull request merged**